polynote-frontend/polynote/main.js

```diff
@@ -240,24 +243,32 @@ const md = require('markdown-it');
       };
   })();
 
-  const MarkdownIt = md({
-      html: true,
-      linkify: false,
-  });
-  
-  MarkdownIt.use(mdk);
+const MarkdownIt = md({
+  html: true,
+  linkify: false,
+});
 
-  window.MarkdownIt = MarkdownIt;
+MarkdownIt.use(mdk);
 
+window.MarkdownIt = MarkdownIt;
 
-  const socket = new SocketSession();
+// set up scala highlighter
```

Comment:
This is some minor changes to the structure of the JS – it turns out that our Scala highlighter wasn't even being included in webpack because it wasn't actually imported. So I guess we've been using the default one, which is broken with `<` and `>`?

Anyway this fixes that.

Reply:
Weird, so basically this import https://github.com/polynote/polynote/blob/master/polynote-frontend/polynote/ui.js#L4 didn't actually do what we expected it to? Does importing in JS not run static code?